src/sentry/models/minidump.py

```diff
@@ -67,10 +75,18 @@ def merge_minidump_event(data, minidump_path):
     context = data.setdefault('contexts', {})
     os = context.setdefault('os', {})
     device = context.setdefault('device', {})
-    os['name'] = info.os_name
-    os['version'] = info.os_version
+    os['type'] = 'os'  # Required by "get_sdk_from_event"
+    os['name'] = _minidump_os_mapping.get(info.os_name, info.os_name)
     device['arch'] = info.cpu_family
 
+    # Breakpad reports the version and build number always in one string,
+    # but a version number is guaranteed even on certain linux distros.
+    match = _version_re.search(info.os_version)
```

Comment:
The version string reported by Breakpad starts with `x.y.z` but then continues with arbitrary garbage depending on the client. We just put it into `"build"` for now, but it doesn't really matter, actually.

 - **macOS**: build number
 - **Windows**: service pack and build number
 - **Linux**: output of `uname -a`